### PR TITLE
Implementation of the `Log Out` button

### PR DESCRIPTION
**Pull request by Oleh-Stefanovych**
Now file page has a `Log Out` button.
image

The `ESLint` result:
image

### CODE REVIEW

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, PTAL.

File: app/presenter/files-presenter/index.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ export class FilesPresenter {
     new FilesPage(root, store)
         .onFolderOpen((targetFolder) => {
           router.redirectTo('/files/' + targetFolder.id);
+        })
+        .onLogOutButtonClick(() => {
+          router.redirectTo('/login');
```
Comment (dkalinin) [inline app/presenter/files-presenter/index.js]: Request should be sent to backend to close session of user. Something like POST /logout.
After that backend will return 403 until user logs in.

**Comment by Oleh-Stefanovych** [comment]
@dkalinin, fixed, PTAL.

**Fixes:**
- Implement simple login logic on development server.
- Implement handling of log out event on UI.

**Note:**
- To start the server, please use `npm run ds`. Application will be available on [http://localhost:5001/](http://localhost:5001/).
- Bug with multiple event subscribing was fixed in #32. After each login, one redundant request is sent more to get the root folder.

The `ESLint` result:
image

File: app/presenter/files-presenter/index.js [hunk 2]
```diff
@@ -36,6 +36,12 @@ export class FilesPresenter {
       }
     });
 
+    this.store.eventBus.subscribe('stateChanged.loggingOutCompleted', (rootFolder) => {
+      if (rootFolder) {
+        router.redirectTo('/login');
```
Comment (dkalinin) [inline app/presenter/files-presenter/index.js]: `rootFolder` is not needed here.

File: app/store/actions.js [hunk 3]
```diff
@@ -189,4 +192,20 @@ export default {
...
+  logOut(store) {
+    return () => {
+      store.commit('setFileListStatus', 'Logging out...');
+      return store.requestServices.sessionRequestService.logOut()
+          .then(() => {
+            store.commit('clearFileListStatus');
+            store.commit('setLoggingOutCompleted');
+          })
+          .catch((responseError) => {
+            store.commit('clearFileListStatus');
```
Comment (dkalinin) [inline app/store/actions.js]: clearFileListStatus can be moved to finally.

**Pull request merged**